ee/modules/600-flant-integration/hooks/license.go

```diff
@@ -14,28 +14,66 @@ import (
 
 	"github.com/flant/addon-operator/pkg/module_manager/go_hook"
 	"github.com/flant/addon-operator/sdk"
+	"github.com/flant/shell-operator/pkg/kube_events_manager/types"
 	"github.com/google/go-containerregistry/pkg/authn"
 	log "github.com/sirupsen/logrus"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+)
+
+const (
+	revokedCMName      = "madison-revoked-project"
+	revokedCMNamespace = "d8-monitoring"
+	revokedCMBinding   = revokedCMName
+
+	globalRegistryPath     = "global.modulesImages.registry"
+	dockerConfigPath       = "/etc/registrysecret/.dockerconfigjson"
+	internalLicenseKeyPath = "flantIntegration.internal.licenseKey"
+	licenseKeyPath         = "flantIntegration.licenseKey"
 )
 
 var _ = sdk.RegisterFunc(&go_hook.HookConfig{
+	Kubernetes: []go_hook.KubernetesConfig{
+		{
+			Name:       revokedCMBinding,
+			ApiVersion: "v1",
+			Kind:       "ConfigMap",
```

Comment:
How can I register in madison again with a new domain? Or what if Prometheus was unarchived? This question is more about how users will know what to do.

Reply:
Documentation is added to config-values.yaml hot to retrieve generated key from internal values.